src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/FindExtensions.cs

```diff
@@ -1,84 +1,47 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Linq.Expressions;
-using System.Reflection;
-using Microsoft.EntityFrameworkCore;
-using Microsoft.EntityFrameworkCore.Metadata;
 
-namespace GovUk.Education.ExploreEducationStatistics.Common.Extensions
+namespace GovUk.Education.ExploreEducationStatistics.Common.Extensions;
```

Comment:
Yeah there weren't any code _changes_, just removals, but I probably still should have clarified. I've just removed `FindAll` and the associated `ContainsMethod` on the advice of Ben because we'd just use LINQ's `Contains` method instead.